runtime/doc/lua.txt

```diff
@@ -543,16 +543,19 @@ Example: File-change detection                                    *watch-file*
     vim.api.nvim_command(
       "command! -nargs=1 Watch call luaeval('watch_file(_A)', expand('<args>'))")
 <
-                                                         *fswatch-limitations*
-When on Linux and using fswatch, you may need to increase the maximum number
-of `inotify` watches and queued events as the default limit can be too low. To
-increase the limit, run: >sh
-    sysctl fs.inotify.max_user_watches=100000
-    sysctl fs.inotify.max_queued_events=100000
+                                                         *inotify-limitations*
+When on Linux and using inotifywait, you may need to increase the maximum
```

Comment:
```suggestion
When on Linux you may need to increase the maximum
```

This applies to both - using inotifywait or the built-in watch mechanism

Reply:
I've fixed it an reformatted the text. Thanks.